lib/puppet/provider/package/pear.rb

```diff
@@ -2,107 +2,98 @@
 
 # PHP PEAR support.
 Puppet::Type.type(:package).provide :pear, parent: Puppet::Provider::Package do
-  desc 'PHP PEAR support. By default uses the installed channels, but you can specify the path to a pear package via ``source``.'
+  desc 'Package management via `pear`.'
 
   has_feature :versionable
   has_feature :upgradeable
   has_feature :install_options
 
-  case Facter.value(:operatingsystem)
-  when 'Solaris'
-    commands pearcmd: '/opt/coolstack/php5/bin/pear'
-  else
-    commands pearcmd: 'pear'
-  end
+  commands pearcmd: 'pear'
```

Comment:
i'd just call this `pear`… you can maybe add a wrapper with default options that you call `pearcmd` (or vice versa?)